front/app/components/IdeaCards/IdeasWithFiltersSidebar.tsx

```diff
@@ -368,14 +368,6 @@ class IdeaCards extends PureComponent<Props & InjectedIntlProps, State> {
     });
   };
 
-  handleDesktopSearchInputClearButtonRef = (element: HTMLButtonElement) => {
-    this.desktopSearchInputClearButton = element;
-  };
-
-  handleMobileSearchInputClearButtonRef = (element: HTMLButtonElement) => {
-    this.mobileSearchInputClearButton = element;
-  };
-
```

Comment:
Not needed anymore with the new SearchInput that uses the IconButton (tested the desktop and mobile searches, not sure why this ref stuff was there).